### PR TITLE
Implement DOMUtils#getDistanceFromTopOfParent

### PR DESCRIPTION
**Pull request by kennyt**
Going to use this instead of `offsetTop` for cross browser compatibility

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @jfurrow

File: src/js/utils/DOMUtils.js [hunk 1]
```diff
@@ -130,6 +130,13 @@ var DOMUtils = {
 
     // We need to also use #contains because the elAtPoint may be a child.
     return el === elAtPoint || el.contains(elAtPoint);
+  },
+
+  getDistanceFromTopOfParent: function (el) {
+    let elTop = el.getBoundingClientRect().top;
+    let parentTop = el.parentNode.getBoundingClientRect().top;
```
Comment (rcorral) [inline src/js/utils/DOMUtils.js]: You should make sure that `el.parentNode` exists, as it won't for `document`

Comment (rcorral) [inline src/js/utils/DOMUtils.js]: and write a good 'ol test.

Comment (kennyt) [inline src/js/utils/DOMUtils.js]: done

**Comment by rcorral** [comment]
Rebase and 🚢

**Pull request merged**